apps/i18n/common/en_us.json

```diff
@@ -195,6 +195,7 @@
   "shareWarningsAge": "Please provide your age below and click OK to continue.",
   "shareWarningsMoreInfo": "More Info",
   "shareWarningsStoreData": "This app built on Code Studio stores data that could be viewed by anyone with this sharing link, so be careful if you are asked to provide personal information.",
+  "showBlock": "Get a block",
```

Comment:
It's certainly not necessary to internationalize it, but I also don't see a huge downside.

I suppose it is the case that the "See hint" string is more likely to have been translated than the "Get a block" string, since the latter is new. That could mean that our test is somewhat contaminated for certain languages, since students will be seeing the two options in two different languages. That does imply to me that we should restrict this to en-us users.

However, I still like having this string here. Having copy defined inline makes me a sad :panda_face:.